src/main/com/cqfn/degitx/simulator/State.kt

```diff
@@ -0,0 +1,8 @@
+package com.cqfn.degitx.simulator
+
+/**
+ * State of physical server or network infrastructure (Edge is the whole way between Node A and Node B)
+ */
+enum class State {
+    ACTIVE, DOWN, SLOW, OUTDATED
```

Comment:
@IlyaMoskva `SLOW` state seems to be non-informative. The speed of node is a compound metric of throughput of network edges and hard drive speed, if simulator controls these both factors independently, we don't need separate `SLOW` state for the node, WDYT?

Reply:
@g4s8 God point, I agree. Those three states could be enough. Fixed.